src/components/CodeInput.tsx

```diff
@@ -10,7 +10,14 @@ export interface CodeInputProps {
 }
 
 export const CodeInput = ({value, onChange, accessibilityLabel}: CodeInputProps) => {
-  const onChangeTrimmed = useCallback(text => onChange(text.trim()), [onChange]);
+  const onChangeTrimmed = useCallback(
```

Comment:
This function is doing more than just trimming the string, the function name `onChangeTrimmed` should be changed to reflect that.